tests/garage/envs/test_normalized_env.py

```diff
@@ -2,11 +2,16 @@
 import unittest
 
 from garage.envs.box2d import CartpoleEnv
-from garage.envs.normalized_env import NormalizedEnv
+from garage.envs.mujoco import SwimmerEnv
+from garage.envs.normalized_env import NormalizedEnv, normalize
 from tests.helpers import step_env
 
 
 class TestNormalizedEnv(unittest.TestCase):
+    def test_can_create_env(self):
+        env = normalize(SwimmerEnv())
```

Comment:
Did this test fail before your PR?

If it failed before your PR, shouldn't `test_pickleable()` have also failed?

Reply:
Yes. I ran it on the tests before the fix, and all 3 tests fail.
After the fix, all of them run fine.

Reply:
Please leave a comment like `# Fixes https://github.com/rlworkgroup/garage/pull/420` inside this test.